collection-scripts/gather

```diff
@@ -80,7 +80,7 @@ Usage: $0 [OPTIONS]
 
 Options:
   -d,  --dr                 Collect DR logs
-  -p,  --provider           Collect logs for provider/consumer cluster
+  -pc, --provider           Collect logs for provider/consumer cluster
```

Comment:
`collecting openshift-storage-client logs from a provider/consumer cluster` please update the description with proper description.